clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -144,9 +144,17 @@ export class SettingsService {
             !SettingsService.hasUssPath();
     }
 
+    /**
+     * Determine if USS copybook files are EBCIDIC or not
+     * @returns true if EBCIDIC
+     */
+    public static isUSSCopybookEbcidic() {
```

Comment:
You have a lot of names for the same thing:

- "paths-uss-encoding" in config
- "Is USS copybook encoding IBM-1047" in config description
- isUSSCopybookEbcidic the function name
- binary: in download options.

Reply:
Agree. Updated except the binary as the ZE API needs this object.